### DISCUSSION TITLE
OSC2025 東京春（2/21-22 リアル展示、セミナー）への出展について

### DISCUSSION THREAD

**Discussion by potappo** [mozilla-japan/translation#798] [General]
前回 #786 

申込は11月下旬からと OSC 運営の Slack にありました。

https://event.ospn.jp/osc2025-spring/

今回の会場は、駒澤大学駒沢キャンパスです。鉄道路線的には東急田園都市線となります。

日程は以下の通りです。

2025年2月21日(金) 10:00～18:00（展示：11:00～17:30）
　　　 2月22日(土) 10:00～17:30（展示：10:00～16:00）

セミナーも現地開催ということですので、過去に行ったようなハンズオン形式のセミナーをやりたいと考えています。人数の集まりを考えると土曜日開催が良いかなと思います。

ブースは今のところ、金曜・土曜の 2 日間（セミナー開催中は一時お休み）出展しようと考えています。この日程であれば、私が年休取れるかと思います。

荷物搬入はどうするか、ネットワークは使えるかなどの情報が出てきたら追記します。
荷物搬入に宅配便が使えるなら、PC はハンズオン用にもう数台 WebDino からお借りして、環境をセットアップしておくことを考えています。

（追記）
展示物の搬入：前日の指定日時で送付可能 金曜から参加なので2/20(木)指定、詳しい指定時間帯や住所は発表後、必要な方に個別に展開
展示物の搬出：ヤマト運輸の着払いにて送付が可能

ネットワーク：無線LANあり
※展示ブースでの記載。セミナー会場に関しても用意するとのこと。

**Comment by potappo** [comment]
ハンズオンセミナーについて
- GitHub とやり取りできるレベルのネットワーク回線さえ確保できれば、GitHub で完結するお手軽ルートで取り組んでもらうのが手っ取り早い（時間の問題もあるので）。
※モバイルルーターを自前で用意しておく必要がありそう。オフラインポータブル環境を USB メモリで配布できないかも考える。

- チューターは過去の実績から 3-4 人程度必要 ※普通サイズの教室が会場であることが前提

**Reply by hmatrjp** [reply]
> お手軽ルートで取り組んでもらうのが

賛成です。もっと深く知りたい方には、動画の案内をするのがいいかもしれません。

ハンズオン用に確保できる時間にもよるのですが、土曜日なら私も参加できるかもです。
（前日21日の夜、名古屋での予定があり、その後に出て間に合うならば。）

事前宅配が使えるなら、グッズに同梱するなどして貸し出せるPCがありますが、WebDINOからの数で十分かもしれませんね。

**Reply by gurezo** [reply]
自分も賛成です。
Wifi 回線については、OSC 事務局にどうなるかを打診しておくのも良いかと思いました。

**Reply by potappo** [reply]
申込が始まりました。1/7(火)までです。12月の忙しさのほどはまだ未定ですが、少なくとも年末年始は帰省するので、できれば今月中に提出しようと思っています。細かいところは 12 月以降のミートアップで相談して詰めていく予定です。

Wifi 回線について、Slack で問い合わせたところ、何らかの形で使えるようになるはず、モバイルルーターの持ち込みも OK とのことでした。
ネットワーク利用不可時の代替策も用意してほしいとのことなので、何かしら考えます。USB メモリですぐコピーできるポータブル環境などが候補です。※手順説明中に回せる程度のコピー時間

申込時に開催日時の希望が出せるのですが、時間帯は以下となっています。いずれも開始時間で、セミナー時間は45分間です。
・2/21(金)：10:00～17:00
・2/22(土)：10:00～15:00 ※16時からライトニングトークのため

提出する希望は以下で考えています。
日付 第1希望：2/21 第2希望：2/22
時間 第1希望：13:00 第2希望：14:00 第3希望：15:00 第4希望：11:00 第5希望：10:00
優先度：日付優先

第4希望以降に16:00以降を書くと金曜確定なので、第4希望は11:00、10:00にしています。
希望時間帯などあればお願いします。

2017年の概要をウェブ検索で見つけてきました。記載内容は似たような感じになると思いますが、「この文言は入れたい」などあればお願いします。
https://www.ospn.jp/osc2017-fall/modules/eguide/e68.html

**Reply by potappo** [reply]
2 月のミートアップはちょうど日付も重なるので、 OSC ハンズオンセミナーに置き換えでかまわないと思っています。
申込ページも OSC でのセミナー概要ページ公開に合わせて公開にしようと思います。
※OSCでのタイムテーブルは1月下旬公開予定とのこと。
あくまでも私の希望なので、2 月中であれば、ミートアップは別の週で別途開催することも可能です（3/1は私は参加不可）。

**Reply by potappo** [reply]
提出時のセミナー内容欄に記載するため、DoorKeeperに イベント URL を確保しました。
https://mozilla.doorkeeper.jp/events/180166
権限があれば中身を観れますが、会場住所を入れたぐらいで、他は後日記載します。

**Reply by hmatrjp** [reply]
@potappo
 > https://mozilla.doorkeeper.jp/events/180166
 
DoorKeeper の方は 2/22 になっていますが、OSC セミナー希望日付の優先順位は
> 日付 第1希望：2/21 第2希望：2/22

で合っていますか？　金曜日のほうが取りやすいのかな？

**Reply by gurezo** [reply]
文面は当時になかった以下の文面をいれるのはいかがでしょう
```
翻訳を始める前にの手順をウェブサイトや動画を使って説明します。
```

文字数に制限がなければ、URL（サイト、動画）を記載するのもありかな？と考えました。

**Reply by gurezo** [reply]
金曜日：学生が多い、社会人少ない
土曜日：学生が多い、社会人？？？
どこらへんをターゲットにしましょうか？（決めの問題ですが
※自分も調整が付けば、金曜日からお手伝いをします。

**Reply by potappo** [reply]
ありがとうございます。第一希望は2/22のつもりでした。金曜土曜どちらが取りやすいかはちょっとわからないので、主観で人が集まりやすそうな土曜で良いかなと思います。金曜の展示ブースでセミナー宣伝もできますし。
ブースは金曜土曜2日で申し込みます。

なお、荷物は前日金曜指定で会場に送付できるとのことです。

**Reply by potappo** [reply]
セミナータイトルは「初心者向け：MDN編集ハンズオン～必要なのはブラウザーだけ！GitHubサイト上だけで完結、お手軽ルート編～」にしようと思います。
現在のシステムで手軽さをアピールするなら、「ブラウザーだけ」をアピールした方が良いかなと思い、こうしました。

セミナー内容は以下で提出しようと思います。遅くても 12/1 夜には出そうと思いますが、何かご意見あればお願いします。
※OSC は外部サイトなので、Web の表記をあえて使っています。出展ブース説明でも使っているので今更ウェブにすることもないかと。

また、OSC での開催のため、若干、対象の間口を拡げて、対象者を「Web 関連の技術に興味をお持ちの方。GitHub でのコントリビューションに興味をお持ちの方。」としようと思います。OSS コントリビューションの観点からすると、GitHub の Pull Request を簡単に体験してもらうという目的としても良いのではないかと思います。ここは追加部分なので、MDN にフォーカスを絞った方が良いなどのご意見があれば、お願いします。

（セミナー内容ここから）
Web 関連の技術ドキュメントである MDN(Mozilla Developer Network) を編集するハンズオンセミナーです。
翻訳作業は敷居が高いと思っている方でも、「てにおは」の修正など、既存文書の簡単な校正から始められる、MDN の編集方法をお伝えします。
なので英語ができなくてもOKです。編集には GitHub のアカウントが必要になります。事前に登録を推奨しますが、当日アカウント取得のサポートも可能です。
作業手順はブラウザーだけで作業する「お手軽ルート」です。GitHub のリポジトリーをクローンして・・・などの本格的な作業環境構築は行いません。最初にスライドによる手順説明を行った上で、コミュニティのチューター役がみなさんをサポートします。予習したい方は以下の動画をご覧ください。
https://youtu.be/KxMmm98emKw
GitHub サイト上でドキュメントのソーステキストを編集し、Pull Request をする形式となります。一番簡単な形式ではありますが、GitHubでのコントリビューションに興味がある方への入門としても最適です。
ブラウザーがあれば作業できますのでスマホだけでも可能ですが、ノートPCやタブレット+キーボードなどの普段文字入力に慣れている端末をご用意ください。
GitHub サイトに接続するための Wifi を用意する予定ですが、やむを得ない事情でネットワークが使えない場合、PC 用ローカル環境配布での作業となります（USB メモリでの配布を想定していますが、若干数、こちらでも導入済み PC を用意します）。
事前に参加者人数を把握したいため、下記の申込ページから登録をお願いします。
https://mozilla.doorkeeper.jp/events/180166

**Reply by potappo** [reply]
申込書は 12/2（月）の朝に提出します。作成は終わっていますが、念のため、朝に再度見直します。
講演者氏名は2017年同様「Mozilla 翻訳コミュニティのメンバー」とする予定です。ハンズオンスタッフは、前回までと同様、複数名の参加を想定しています。
直近の福岡を見る限り、割と自由そうなのでそれで大丈夫そうですが、それで通らなかった場合は、私と確実に参加出来そうな方の氏名の連名で提出します。ただのセミナーではなくハンズオンセミナーであることを強調するには複数名の方が良いかと思います。

**Comment by potappo** [comment]
今日のミートアップで話したことの共有です。

・申込は完了
・OSC 公式のセミナープログラム公開は 1/23(木）予定なので、それに合わせて DoorKeeper のセミナー申込ページも公開予定。
・OSC 説明スライド作成中（クリスマス・忘年会期間前に済ませたいので 12/22,23完成目標）

セミナー用スライドは以下の URL です。まだテキストだけです。明日以降、いくつか操作画面スクリーンショットを足す予定です。
https://docs.google.com/presentation/d/12Vi0yeu_l4IaazBIS6EW6GevMNIQjujAMfRtfRdKBsM/edit#slide=id.p

**Reply by potappo** [reply]
セミナー用スライドに画像を追加しました。
明日の朝に OSC 事務局の方に講演資料として URL を連絡します。
当日はネットワーク使えれば URL からそのまま投影、使えない場合に備えて PDF 形式などでダウンロードしておきます。
※完全にネットワークが使えなかった場合用の資料も用意する必要がありますが、それは年末年始以降に作成します

**Reply by gurezo** [reply]
@potappo 
スライド作成ありがとうございます。

大学の教室を考えると 2P の URL とは別に QR を配置して置くと良いと思いました。
ぱっと見気になったのはこれくらいでした。

**Reply by hmatrjp** [reply]
スライド作成ありがとうございます。 @potappo 
問題ないと思います。

p.3 の
>  ・記事を探す
の部分、状況としてよくありそうなのは
・記事を探す、または、記事を読んでいて直したい部分を見つけた
かもしれませんね。

**Comment by potappo** [comment]
今日のミートアップで話したことのメモ＋アルファです。

・当日の参加者に貸し出す端末 は、今借りているデモ用 PCとぐれさん提供の PC・タブレットとする 
   ※セミナー資料を会場スクリーンに投影するための PC は私自身の ノート PC を持参します。当日プレゼン内容を書き換えたりとかもあるかもしれないので。
・2日間なので、チラシ、グッズは 1.5 倍用意する。送付先は OSC 事務局からの展開待ち。事務局からのメール見返したところ、1/31 に詳細な出展者向け案内が来るようです。
・確定したセミナー日時は 1/23(木) 公開予定
・次回の翻訳ミートアップオンライン開催は、3/22(土) を予定、2 月分は OSC のセミナーで代替。

何かあれば随時こちらで報告、連絡します。

**Reply by potappo** [reply]
Slack で報告しましたが、記録として、こちらにも。
DoorKeeper の申込ページを公開しました。
開催日時は 2/22(土) 13:00-13:45です。
https://mozilla.doorkeeper.jp/events/180166

**Reply by gurezo** [reply]
@potappo 
手配ありがとうございます！

**Comment by potappo** [comment]
OSC 事務局から出展者向け案内が来ました。
セキュリティ上の観点から案内をパブリックな場所に展開はしませんので、欲しい方は、OSC の Slack で入手お願いします。

出展者向け案内にある当日の情報として、昼食情報を書いておきます。これは別にバブリックでかまわないと思いますので。
学内の食堂は OSC 関係者・来場者は利用不可、学内のコンビニは金曜 14 時まで営業とのことです。
学食内とコンビニ前スペースで持込飲食が可能とのことです（スペースの規模は不明）。
会場の立地的には会場周辺にもコンビニや食べるところは普通にありそうかと思います。

@hmatrjp
OSC会場の荷物送付先住所を Slack DM で送りましたので、確認をお願いします。

**Reply by hmatrjp** [reply]
荷物の宛先住所と注意事項読みました。ありがとうございます。 @potappo

**Reply by gurezo** [reply]
@potappo 
お手すき時に自分にも住所のDM 下さい
パソコン類を送ろうと思います！

**Reply by potappo** [reply]
@gurezo 送りましたので確認ください。

**Reply by hmatrjp** [reply]
@potappo @gurezo 
会場向けに、ノベルティグッズを発送しました。 ゆうパック 7522-3814-7183 です。
品目は写真を、数は[一覧表](https://docs.google.com/spreadsheets/d/1N73mciqvlQbuC4457OMteS-TXDldj75XybDKfK8CToI/edit?pli=1&gid=345022493#gid=345022493)のI列をご参照ください。なお、キャップは4つ入れてあって、スタッフ用の想定です。

箱の底に同梱したPCは、ハンズオンで不足したときにと思っていますが、1日めも何かの役に立つなら使ってください。
1日め終了後は会場放置で構いません。（テーブル垂れ幕の下とか？）　OSCguestユーザーだけがあり、パスワードはoscです。

![swag20250219](https://github.com/user-attachments/assets/9f03ba6a-28f5-479f-9d10-401a0c4caf93)

**Reply by hmatrjp** [reply]
（さっきの投稿に書き忘れました。）私自身は2/22（土）朝に自宅を出発し、12時前後に会場着予定です。
帰りは、ライトニングトークまで聞いて会場を出発したら間に合いそうな列車を予約してあります。

**Reply by potappo** [reply]
@hmatrjp グッズ発送ありがとうございます。

チラシはセミナーの宣伝を1行足した版を井上さんに印刷依頼して、こちらも発送済の連絡を受けています。

後は展示用PCのアップデートを昨日からやっていたりと、最終準備中です。

**Reply by gurezo** [reply]
@potappo @hmatrjp 
セミナー用 PC は本日発送しました。
※ @potappo さんには送り状を DM しました。

02/22(土)は、開場に合わせて会場入りする予定です。

**Comment by hmatrjp** [comment]
@potappo @gurezo 
手持ちのデバイスで slack に入れないことが発覚、気づいてもらえればラッキーということで、こちらにメッセージします。
12時過ぎに現地着予定です。

**Comment by potappo** [comment]
無事に 2 日間の出展が終了しました。お疲れ様でした。

1 日目は一人ブースだと大変なところが若干あったので、次回以降の平日参加は確実に 2 人以上参加できる前提にしたいと思います。わからなければ 2 日目のみという感じで良いかと思います。

2 日目はセミナーが無事にみなさん Pull Request できるところまで終わって良かったです。
（若干タイムオーバーしましたが）
進行に手間取ったところなど反省点もありますが、実際にやった経験で掴んだ感覚を活かして、次回の進行をブラッシュアップできればと思います。

次回は 5/31(土) 名古屋の参加を予定しています。3 月になったら申込が始まるかと思いますので、ディスカッションもそのとき、もしくは、3 月のミートアップ開催時に作成する予定です。
